pkgs/build-package.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -x
+
+sudo chown -R build:build /workdir/pkgs
+
+PIKAUR_CMD="PKGDEST=/workdir/pkgs pikaur --noconfirm -S -P /workdir/${1}/PKGBUILD"
+PIKAUR_RUN=(bash -c "${PIKAUR_CMD}")
+"${PIKAUR_RUN[@]}"
```

Comment:
Might be worth migrating all to makepkg instead of relying on pikaur. Can be tedious but can be simply scripted and reused if done right.